src/routes/safe/components/Apps/components/GasEstimationInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+import Img from 'src/components/layout/Img'
+import CheckIcon from 'src/assets/icons/check.svg'
+import AlertIcon from 'src/assets/icons/alert.svg'
+
+type OwnProps = {
+  appEstimation: number
+  internalEstimation: number
+  loading: boolean
+}
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+const imgStyles = {
+  marginRight: '5px',
+}
+
+const GasEstimationInfo = ({ appEstimation, internalEstimation, loading }: OwnProps): React.ReactElement => {
+  if (loading) {
+    return <p>Loading...</p>
```

Comment:
I would probably not show this 🤔 or use a different message (e.g. Checking transaction parameters)